afkak/kafkacodec.py

```diff
@@ -384,7 +385,7 @@ def decode_offset_response(cls, data):
                     relative_unpack('>ihi', data, cur)
 
                 offsets = []
-                for _j in range(num_offsets):
+                for _i in range(num_offsets):
```

Comment:
Eww.  I guess since nothing further down in the enclosing loop(s) references the outer _i's it's fine, but ewww :-)